kernel/src/actions/visitors.rs

```diff
@@ -278,8 +278,9 @@ mod tests {
     use super::*;
     use crate::{
         actions::schemas::log_schema,
+        client::arrow_data::ArrowEngineData,
```

Comment:
rust newbie question: From what I can see, this arrow dep is only needed by tests. Does the test dep make arrow a prod dep as well? Or does rust module keep them separate?

Reply:
You put them under `[dev-dependencies]` for tests and the `#[cfg(test)]` to only build in test .